Libs/MRML/Core/vtkMRMLClipModelsNode.h

```diff
@@ -90,6 +90,19 @@ class VTK_MRML_EXPORT vtkMRMLClipModelsNode : public vtkMRMLNode
       ClipNegativeSpace = 2,
     };
 
+  ///
+  ///Indicates what clipping method should be used
+  ///Straight cut, whole cell extraction, or whole cell extraction with boundary cells
+  vtkGetMacro(ClippingMethod, int);
+  vtkSetMacro(ClippingMethod, int);
+
+  enum
+    {
+      Straight = 0,
+      WholeCells = 1,
+      WholeCellsWithBoundary = 2,
+    };
```

Comment:
Could you name the enum (ClippingMethodType for example) and follow the indentation style like here: https://github.com/Slicer/Slicer/blob/master/Libs/MRML/Core/vtkMRMLModelNode.h#L118